onnxruntime/core/providers/coreml/model/model.mm

```diff
@@ -431,6 +454,47 @@ - (Status)predict:(const std::unordered_map<std::string, OnnxTensorData>&)inputs
 namespace onnxruntime {
 namespace coreml {
 
+Status GetMLMultiArrayCopyInfo(const MLMultiArray* array, int64_t* num_blocks, int64_t* block_size, int64_t* stride) {
```

Comment:
can we use `int64_t&`? I guess we should keep the objc type a pointer. also, maybe `array` can be marked with `_Nonnull` as it is outside of the NS_ASSUME_NONNULL section.